src/TypeScript.UnitTests/NodeJSLocator/CompatibleNodeLocatorTests.cs

```diff
@@ -52,6 +52,21 @@ public void Locate_NoDetectedVersions_Null()
             result.Should().BeNull();
         }
 
+        [TestMethod]
+        public void Locate_NoCompatibleVersions_ReturnsNull()
```

Comment:
There are tests for "no _detected_ versions" and "first compatible version", but not for "no compatible versions".